content/examples/grid-css-grid.mdx

```diff
@@ -1,5 +1,37 @@
 ---
 title: Grid - CSS Grid
-lastUpdatedDate: 2022-04-06T12:00:00.000Z
+lastUpdatedDate: 2022-04-27T12:00:00.000Z
 ---
-Grid - CSS Grid
\ No newline at end of file
+
+<Section heading="Usage:" width="full" className="su-max-w-1200">
+  ```html
+  <div className="su-grid md:su-grid-cols-3 su-grid-gap">
+    <div className="...">01</div>
+    <div className="...">02</div>
+    <div className="...">03</div>
+  </div>
+  ```
+
+  <div className="su-grid md:su-grid-cols-3 su-grid-gap">
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white">01</div>
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white">02</div>
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white">03</div>
+  </div>
+</Section>
+
+<Section heading="Page layout example:" width="full" className="su-max-w-1200">
+  ```html
+  <div className="su-grid md:su-grid-cols-4 su-gap">
+    <div className="md:su-col-start-1 md:su-col-end-5">Header</div>
+    <div className="md:su-col-start-1 md:su-col-end-2">Sidebar</div>
+    <div className="md:su-col-start-2 md:su-col-end-5">Main Content</div>
+    <div className="md:su-col-start-1 md:su-col-end-5">Footer</div>
+  </div>
+  ```
+  <div className="su-grid md:su-grid-cols-4 su-grid-gap">
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white md:su-col-start-1 md:su-col-end-5">Header</div>
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white md:su-col-start-1 md:su-col-end-2">Sidebar</div>
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white md:su-col-start-2 md:su-col-end-5">Main Content</div>
+    <div className="su-p-4 su-rounded-lg su-shadow-lg su-bg-digital-blue su-text-white md:su-col-start-1 md:su-col-end-5">Footer</div>
```

Comment:
These works but if we could simplify the grid classes a bit, that'd be awesome @Lyafel 😄 

For the header and footer, let's use `md:su-col-span-4` instead of `md:su-col-start-1 md:su-col-end-5`
For the sidebar, I believe you can remove md:su-col-start-1 md:su-col-end-2 since the default is spanning 1 column, if that doesn't work, you could add md:su-col-span-1
For the main content, please use `md:su-col-span-3` instead of `md:su-col-start-2 md:su-col-end-5`